src/tests/system/tests/test_ldap_extra_attrs.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.topology import KnownTopologyGroup
+
+
+@pytest.mark.tier(1)
+@pytest.mark.ticket(gh=4153, bz=1362023)
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_ldap_extra_attrs__contains_mail(client: Client, provider: GenericProvider):
+    """
+    :title: SSSD fails to start when ldap_user_extra_attrs contains mail
+    :setup:
+        1. Add mail to ldap_user_extra_attrs
+    :steps:
+        1. Start SSSD
+    :expectedresults:
+        1. SSSD starts successfully
+    :customerscenario: True
+    """
+    client.sssd.domain["ldap_user_extra_attrs"] = "mail, firstname:givenname, lastname:sn"
+
+    try:
+        client.sssd.start()
+    except Exception:
+        pytest.fail("Exception shouldn't be raised")
```

Comment:
provide information about the exception, eg

pytest.fail(f"Exception shouldn't be raised but we got {type(e)}: str(e)")